From db33a124a0e45c511ee45192da3c191364715e18 Mon Sep 17 00:00:00 2001
From: Ikey Doherty <ikey@evolve-os.com>
Date: Mon, 9 Mar 2015 21:51:10 +0000
Subject: [PATCH 3/3] session: XDG_CURRENT_DESKTOP is case sensitive. Whoda
 thunk it

This fixes issues within Budgie Menu whereby we shot ourselves in the foot
and filtered our menu items we actually wanted
---
 session/BudgieSession.vala | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/session/BudgieSession.vala b/session/BudgieSession.vala
index 7cb0b23..1caa56e 100644
--- a/session/BudgieSession.vala
+++ b/session/BudgieSession.vala
@@ -74,7 +74,7 @@ public class Session : GLib.Application
 
         // Make Qt, etc, happy
         if (Environment.get_variable("XDG_CURRENT_DESKTOP") == null) {
-            Environment.set_variable("XDG_CURRENT_DESKTOP", "gnome", true);
+            Environment.set_variable("XDG_CURRENT_DESKTOP", "GNOME", true);
         }
 
         /* Make Java, Google Chrome, and xdg-open happy.
-- 
1.8.4

